src/stories/molecules/pricing-summary.stories.svelte

```diff
@@ -1,35 +1,116 @@
-<script module>
+<script module lang="ts">
   import { brandingModes } from "../../../.storybook/modes";
   import { defineMeta, setTemplate } from "@storybook/addon-svelte-csf";
   import { renderInsideNavbarBody } from "../decorators/layout-decorators";
   import PricingSummary from "../../ui/molecules/pricing-summary.svelte";
   import {
-    product,
+    priceBreakdownTaxDisabled,
+    priceBreakdownTaxDisabledIntroPricePaidUpfront,
+    priceBreakdownTaxDisabledIntroPriceRecurring,
     subscriptionOption,
-    subscriptionOptionWithTrial,
     subscriptionOptionWithIntroPricePaidUpfront,
     subscriptionOptionWithIntroPriceRecurring,
+    subscriptionOptionWithTrial,
     subscriptionOptionWithTrialAndIntroPricePaidUpfront,
     subscriptionOptionWithTrialAndIntroPriceRecurring,
-    nonSubscriptionOption,
-    consumableProduct,
-    priceBreakdownTaxDisabled,
-    priceBreakdownTaxDisabledIntroPriceRecurring,
-    priceBreakdownTaxDisabledIntroPricePaidUpfront,
   } from "../fixtures";
 
+  import { parseISODuration } from "../../helpers/duration-helper";
+  import type { PricingPhase } from "../../entities/offerings";
+
+  const billingDurations = ["P1W", "P1M", "P3M", "P6M", "P1Y", null];
+  const introDurations = ["P1W", "P1M", "P3M", "P6M", "P1Y", null];
+  const trialDurations = ["P3D", "P1W", "P2W", "P1M", null];
```

Comment:
These aren't exhaustive, but not sure if they're intended to be or not

Reply:
Yeah, they are just to test some scenarios, also @iga-gawronska will continue working on this component so we can definitely add more later.